quipucords/api/aggregate_report/view.py

```diff
@@ -1,12 +1,67 @@
 """View for aggregate report."""
 
+from drf_spectacular.utils import extend_schema
+from rest_framework import serializers
 from rest_framework.decorators import api_view
 from rest_framework.exceptions import NotFound
 from rest_framework.response import Response
 
 from api.aggregate_report.model import get_aggregate_report_by_report_id
 
 
+class ResultsSerializer(serializers.Serializer):
```

Comment:
please move all these serializers to a dedicated `serializer` module to conform with the pattern used across other models.